sycl/test-e2e/FreeFunctionKernels/accessor_as_kernel_parameter.cpp

```diff
@@ -0,0 +1,80 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// This test verifies whether sycl::accessor can be used with free function
+// kernels extension.
+
+#include <sycl/ext/oneapi/free_function_queries.hpp>
+
+#include "helpers.hpp"
+
+SYCL_EXT_ONEAPI_FUNCTION_PROPERTY((syclexp::single_task_kernel))
+void globalScopeSingleFreeFunc(sycl::accessor<int, 1> Accessor,
+                               size_t NumOfElements, int Value) {
+  for (size_t i = 0; i < NumOfElements; ++i) {
+    Accessor[i] = Value;
```

Comment:
Nit; Inconsistent variable name.
```suggestion
  for (size_t I = 0; I < NumOfElements; ++I) {
    Accessor[I] = Value;
```

Reply:
https://github.com/intel/llvm/pull/18672/commits/6028b4513205ef9ece009604ee8513abdfef87df